UI/style.css

```diff
@@ -0,0 +1,198 @@
+img,
```

Comment:
Please remove the UI files from the PR on Task 4, they do not belong here

Reply:
Done.